skiko/build.gradle.kts

```diff
@@ -1068,12 +1104,13 @@ publishing {
         allJvmRuntimeJars.forEach { entry ->
             val os = entry.key.first
             val arch = entry.key.second
-            create<MavenPublication>("skikoJvmRuntime${targetId(os, arch)}") {
+            create<MavenPublication>("skikoJvmRuntime${toTitleCase(os.id)}${toTitleCase(arch.id)}") {
```

Comment:
Not sure if I like this particular change. Readability seems better when target is clearly marked, and this captializing just makes readability worse.

Reply:
What's the point of readability, when different cases are mixed in inconsistent and unpredictable way?
If we could change the default case format for all tasks, I would be pro dash case.
But that's not possible, and lack of some predictable naming convention and style seriously bothers me

Reply:
OK